test/application_system_test_case.rb

```diff
@@ -7,45 +7,52 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
 
   # https://web.archive.org/web/20170730200309/http://blog.paulrugelhiatt.com/rails/testing/capybara/dropzonejs/2014/12/29/test-dropzonejs-file-uploads-with-capybara.html
   def drop_in_dropzone(file_path, dropzoneSelector)
+    existing_inputs = page.all('.fakeFileInput')
```

Comment:
I rewrote this test so that multiple file inputs could be inserted onto a page.

I did notice that this slows down the test considerably, I'm sure this could be written to be more efficient.

I'll put that on my to-dos (will have to brush up on Ruby some more)